src/app/images/components/ImagesTable/DownloadImages/DownloadImages.tsx

```diff
@@ -202,7 +202,7 @@ const DownloadImages: React.FC = () => {
 
   return (
     <>
-      <p>
+      <p data-testid="download-images-form">
```

Comment:
Is this test ID necessary? We have a custom matcher `getByTextContent` that should work for getting text that's broken up across multiple lines/elements